integration_tests/src/main/python/map_test.py

```diff
@@ -192,7 +192,7 @@ def query_map_scalar(spark):
 
 
 @allow_non_gpu('WindowLocalExec')
-@datagen_overrides(seed=0, reason='https://github.com/NVIDIA/spark-rapids/issues/9683')
+@datagen_overrides(condition=is_before_spark_340(), seed=0, reason="https://issues.apache.org/jira/browse/SPARK-40089")
```

Comment:
the spark bug you pointed to was fixed in 3.3.1 and 3.2.3 as well so is this correct?  I guess its ok but aren't we missing testing 3.3.2 and 3.2.3 with different seeds?

Reply:
Yes, I got lazy and took the highest version of Spark 3.4.0 for which the bug was fixed. 

It's just going to make the condition more complicated